From 3c22ed20ae0cab20bacda55e100349d8925c420a Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 12 Sep 2014 14:29:45 -0400
Subject: [net] openvswitch: Added (unsigned long long) cast in printf

Message-id: <50cf4283e436a4a063868136ff06fb237084e3b3.1410531783.git.jbenc@redhat.com>
Patchwork-id: 94019
O-Subject: [RHEL7.1 net PATCH 12/50] openvswitch: Added (unsigned long long) cast in printf
Bugzilla: 1110384
RH-Acked-by: Flavio Leitner <fbl@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110384
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7948291

commit cc23ebf3bb4348fb022c0d25494307459bb2e539
Author: Daniele Di Proietto <daniele.di.proietto@gmail.com>
Date:   Mon Feb 3 14:09:01 2014 -0800

    openvswitch: Added (unsigned long long) cast in printf

    This is necessary, since u64 is not unsigned long long
    in all architectures: u64 could be also uint64_t.

    Signed-off-by: Daniele Di Proietto <daniele.di.proietto@gmail.com>
    Signed-off-by: Jesse Gross <jesse@nicira.com>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/openvswitch/flow_netlink.c b/net/openvswitch/flow_netlink.c
index 5517bd6..1b22ad2 100644
--- a/net/openvswitch/flow_netlink.c
+++ b/net/openvswitch/flow_netlink.c
@@ -216,14 +216,14 @@ static bool match_validate(const struct sw_flow_match *match,
  if ((key_attrs & key_expected) != key_expected) {
   /* Key attributes check failed. */
   OVS_NLERR("Missing expected key attributes (key_attrs=%llx, expected=%llx).\n",
-    key_attrs, key_expected);
+    (unsigned long long)key_attrs, (unsigned long long)key_expected);
   return false;
  }
 
  if ((mask_attrs & mask_allowed) != mask_attrs) {
   /* Mask attributes check failed. */
   OVS_NLERR("Contain more than allowed mask fields (mask_attrs=%llx, mask_allowed=%llx).\n",
-    mask_attrs, mask_allowed);
+    (unsigned long long)mask_attrs, (unsigned long long)mask_allowed);
   return false;
  }
 
-- 
1.7.1